dwave_networkx/generators/zephyr.py

```diff
@@ -574,53 +592,58 @@ def zephyr_sublattice_mappings(source, target, offset_list=None):
-    isomorphic to the source graph. However, if the target graph is not of
-    perfect yield, these functions do not generally produce isomorphisms (for
+    This sublattice mapping is used to identify subgraphs of the target Zephyr graph 
+    which is isomorphic to the source graph. However, if the target graph is not of
+    perfect yield,\ [#]_ this function does not generally produce isomorphisms; for
     example, if a node is missing in the target graph, it may still appear in
-    the image of the source graph).
+    the image of the source graph.
 
-    Note that the tile parameter of Chimera graphs must be either the
-    same or double that of the target Zephyr graphs; if both graphs are
-    Zephyr graphs, the tile parameters must be the same. The mappings
-    produced preserve the linear ordering of tile indices; see the
+    The tile parameter of Chimera graphs must be either the same or double 
+    that of the target Zephyr graphs; if both graphs are Zephyr graphs, 
+    the tile parameters must be the same. The mappings produced preserve 
+    the linear ordering of tile indices; see the
     ``_zephyr_zephyr_sublattice_mapping``,
     ``_double_chimera_zephyr_sublattice_mapping``, and
-    ``_single_chimera_zephyr_sublattice_mapping`` internal functions for more
-    details.
-
-    Academic note: the full group of isomorphisms of a Chimera graph includes
-    mappings which permute tile indices on a per-row and per-column basis, in
-    addition to reflections and rotations of the grid of unit tiles where
-    rotations by 90 and 270 degrees induce a change in orientation.  The
-    isomorphisms of Zephyr graphs permit permutations of major tile indices on a
-    per-row and per-column basis, in addition to reflections of the grid which
-    induce inversion of orthogonal minor offsets, and rotations which induce
-    inversions of minor offsets and/or orientation. The full set of sublattice
-    mappings would take those isomorphisms into account; we do not undertake
-    that complexity here.
-
+    ``_single_chimera_zephyr_sublattice_mapping`` internal functions.
+    
+    .. [#]
+        The yield is the percentage of working qubits on a QPU and the subset 
+        of available qubits is called
+        the `working graph <https://docs.dwavesys.com/docs/latest/c_gs_4.html#the-working-graph>`_.
+        
     Parameters
     ----------
         source : NetworkX Graph
             The Chimera or Zephyr graph that nodes are input from.
         target : NetworkX Graph
             The Zephyr graph that nodes are output to.
         offset_list : iterable (tuple), optional (default None)
-            An iterable of offsets. This can be used to reconstruct a set of
-            mappings, as the offset used to generate a single mapping is stored
+            An iterable of offsets that can be used to reconstruct a set of
+            mappings since the offset used to generate a single mapping is stored
```

Comment:
```suggestion
            mappings. The offset used to generate a single mapping is stored
```

Here and in the other topologies: a 32-word sentence of this complexity is stretching it a bit.